commit e58d64a16abc2304c4dcb644411eb9580bf63b1e
Author: Marc-André Lureau <marcandre.lureau@redhat.com>
Date:   Tue May 15 17:30:39 2018 +0200

    ccid-card-passthru: fix regression in realize()
    
    Since cc847bfd16d894fd8c1a2ce25f31772f6cdbbc74, CCID card-passthru
    fails to intialize, because it changed a debug line to an error,
    probably by mistake. Change it back to a DPRINTF debug.
    
    (solves Boxes creating VM with smartcard passthru failing to start)
    
    Signed-off-by: Marc-André Lureau <marcandre.lureau@redhat.com>
    Reviewed-by: Philippe Mathieu-Daudé <f4bug@amsat.org>
    Message-id: 20180515153039.27514-1-marcandre.lureau@redhat.com
    Signed-off-by: Gerd Hoffmann <kraxel@redhat.com>

diff --git a/hw/usb/ccid-card-passthru.c b/hw/usb/ccid-card-passthru.c
index 7684db0cb3..25fb19b0d7 100644
--- a/hw/usb/ccid-card-passthru.c
+++ b/hw/usb/ccid-card-passthru.c
@@ -345,7 +345,7 @@ static void passthru_realize(CCIDCardState *base, Error **errp)
     card->vscard_in_pos = 0;
     card->vscard_in_hdr = 0;
     if (qemu_chr_fe_backend_connected(&card->cs)) {
-        error_setg(errp, "ccid-card-passthru: initing chardev");
+        DPRINTF(card, D_INFO, "ccid-card-passthru: initing chardev");
         qemu_chr_fe_set_handlers(&card->cs,
             ccid_card_vscard_can_read,
             ccid_card_vscard_read,
